src/components/Messages/CreateMessageModal/SearchBarMessages.tsx

```diff
@@ -0,0 +1,157 @@
+import SearchRoundedIcon from "@mui/icons-material/SearchRounded";
+import {
+  Autocomplete,
+  Avatar,
+  Box,
+  IconButton,
+  InputAdornment,
+  List,
+  ListItemAvatar,
+  ListItemText,
+  Stack,
+  TextField,
+  Typography,
+} from "@mui/material";
+import { OtherUser } from "./MessagesModalList";
+import { useAppSelector } from "../../../state/hooks";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+
+const styles = {
+  box: {
+    margin: "auto",
+    alignItems: "center",
+    justifyContent: "center",
+    padding: 1,
+    width: "95%",
+  },
+  searchField: {
+    "& .MuiOutlinedInput-root": {
+      borderRadius: "50px",
+    },
+  },
+  avatar: { margin: "auto" },
+  displayName: {
+    fontWeight: "bold",
+  },
+  primaryTextContainer: {
+    gap: 0.5,
+    width: "30%",
+    height: "100%",
+  },
+  stack: {
+    width: "100%",
+  },
+  autocomplete: { "	.MuiAutocomplete-input": { paddingLeft: 0 } },
```

Comment:
Can you please remove the extra spaces? Also I think you need the "&" in front of the period right?